index.html

```diff
@@ -508,6 +508,8 @@ <h4>Computation steps</h4>
                 </li>
                 <li id="comp_name_from_content_for_each_child"><span id="step2F.iii"><!-- Don't link to this legacy numbered ID. --></span><em>Name From Each Child:</em> For each <code>rendered child node</code> of the <code>current node</code>:
                   <ol>
+                    <li id="comp_name_from_content_find_child_actions">If the <code>current node</code> has an <code>aria-actions</code> [=attribute=] with an IDREF matching the rendered child node, skip steps b-d for this child node.</li>
+                    <li>If the <code>current node</code> has an <code>aria-actions</code> [=attribute=] with at least one valid IDREF matching any of the descendents of the rendered child node, exclude the referenced descendants from the name from content of the <code>current node</code>.</li>
```

Comment:
1. Can't use `IDREF` like this since it excludes the static element array plan to support references across document boundaries... (e.g. `el.ariaActionsElements`) 
2. For both clarity and longevity, we shouldn't reference steps numberings like "b–d"… Reference each step by name if needed.
3. This step seems like it would be a little more complicated than the proposed wording. For example, it's missing a reference to the current accname traversal. We may need to include all child nodes and then loop through with a set of exclusions (contained action reference being one of many) that remove them individually from the child nodes list. (I'm sorry I don't have a specific working suggestion.)